reference-adapter/mgramseva-ifix-adapter/src/main/java/org/egov/ifix/service/PspclEventService.java

```diff
@@ -0,0 +1,11 @@
+package org.egov.ifix.service;
+
+import org.egov.ifix.models.mgramseva.ChallanResponseDTO;
+
+import java.util.Optional;
+
+public interface PspclEventService {
+    void processPspclEventForMgramseva(String eventType);
+
+    Optional<ChallanResponseDTO> getOldestChallanByBillDate(String tenantId);
```

Comment:
After deciding to use the billing service to get a consolidated bill, do we need this anymore?

Reply:
Just keep it for future reference.

Reply:
sure.

Reply:
done